cypress/integration/seededFlows/articleFlows/postModerationTools.spec.js

```diff
@@ -40,4 +40,24 @@ describe('Moderation Tools for Posts', () => {
       cy.findByRole('button', { name: 'Moderation' }).should('not.exist');
     });
   });
+
+  it('should not alter tags from a post if a reason is not specified', () => {
```

Comment:
The two newly added tests are checking for the same thing, but one is in the context of the mod page and one in the context of the article page.